Resources/Prototypes/GameRules/events.yml

```diff
@@ -182,6 +182,10 @@
   noSpawn: true
   components:
   - type: StationEvent
+    startAnnouncement: station-event-vent-creatures-start-announcement
+    startAudio:
+      path: /Audio/Announcements/attention.ogg
+    startDelay: 10
     earliestStart: 30
```

Comment:
Firefox lagged but I was gonna say would be better to make these use composition but not a dealbreaker.

Reply:
Oh god code talk. What's composition exactly?